lib/game.rb

```diff
@@ -97,30 +100,46 @@ def bot_turn_sequence
       @board.print_layout
   end
   
-  def game_sequence
+  def game_logic
       player_1_turn_sequence
-      return (print "\n#{@board.player_1} wins!") if @player_1_turn.connect_four == @board.player_1
+      return (print "\n#{@board.player_1} wins!\n") if @player_1_turn.connect_four == @board.player_1
       bot_turn_sequence
-      return (print "\n#{@board.player_2} wins!") if @player_2_turn.connect_four == @board.player_2
+      return (print "\n#{@board.player_2} wins!\n") if @player_2_turn.connect_four == @board.player_2
   end
 
 
   #Switched player number/name references to @board ones - done
   #Restricted options in column select through loop - done
   #Need to make column selection not case sensitive - not yet
   #Need to make bot play again if selected full column - not yet
+  #add draw condition and print
+  #game returns user back to main menu - done
+  #Fixed bug where user could not type p and still proceed to play game.
```

Comment:
We need a way to exit the game if we don't want to finish playing!! I couldn't exit the program when I was testing it.